docs/en/observability/aws-deploy-elastic-serverless-forwarder.asciidoc

```diff
@@ -177,6 +223,15 @@ For `elasticsearch` the following arguments are supported:
   * `args.batch_max_bytes`: (Optional) Maximum size in bytes to send in a single bulk request. Default value: 10485760 (10MB).
   * `args.ssl_assert_fingerprint`: (Optional) SSL fingerprint for self-signed SSL certificate on HTTPS transport.
 
+For `logstash` the following arguments are supported:
+
+  * `args.logstash_url` URL of elasticsearch endpoint in the format `http(s)://host:port`
+  * `args.username`: (Optional) Username of the logstash instance to connect to. Mandatory if HTTP Basic authentication is enabled in Logstash.
+  * `args.password`: (Optional) Password of the logstash instance to connect to. Mandatory if HTTP Basic authentication is enabled in Logstash.
+  * `args.max_batch_size`: (Optional) Maximum number of events to send in a single HTTP(s) request. Default value: 500
+  * `args.compression_level`: (Optional) The GZIP compression level for HTTP(s) requests towards logstash. It can be any integer value between 1 (minimum compression, best performance, highest amount of bytes sent) and 9 (maximum compression, worst performance, lowest amount of bytes sent). Default value: 1
```

Comment:
```suggestion
  * `args.compression_level`: (Optional) The GZIP compression level for HTTP(s) requests towards {ls}. It can be any integer value between 1 (minimum compression, best performance, highest amount of bytes sent) and 9 (maximum compression, worst performance, lowest amount of bytes sent). Default value: 1
```